snap/snapcraft.yaml

```diff
@@ -23,17 +23,15 @@ license: MIT
 parts:
   git-machete:
     plugin: python
-    python-version: python3
     source: .
     # We're setting the version via scriptlet rather than using `version: git`
     # since only pushing commits to master (and not pushing tags) triggers Snapcraft builds.
-    override-pull: |
+    override-build: |
       set -x
-      snapcraftctl pull
-      version=$(grep "__version__ = " git_machete/__init__.py | cut -d\' -f2)
+      snapcraftctl build
+      version="$(python3 setup.py --version)"
       snapcraftctl set-version "$version"
 
 apps:
   git-machete:
     command: bin/git-machete
-    adapter: full
```

Comment:
Without that... can the installed snap access network or files in the home directory? 🤔

Reply:
`Adapter's` default value is `full`, there is no need for this anymore

Reply:
Okay! didn't remember that